Main/Application.cpp

```diff
@@ -1113,6 +1127,23 @@ static int lSetGaugeColor(lua_State* L /*int colorIndex, int r, int g, int b*/)
 	return 0;
 }
 
+int lInclude(lua_State* L)
```

Comment:
Doesn't lua already have `require`?
https://www.lua.org/pil/8.1.html

Reply:
At the time I wasn't aware of `LUA_PATH` as anything more than an environment variable. I'll look at using that to configure `require` and see where it goes.